docs/content/_index.md

```diff
@@ -28,7 +28,7 @@ Athens is easy to run yourself. We give you a few options:
 - You can run it as a [Docker](https://www.docker.com/) image (see [here](./install/shared-team-instance/) for how to do that)
 - You can run it on [Kubernetes](https://kubernetes.io) (see [here](./install/install-on-kubernetes/) for how to do that)
 
-We also run an experimental server for public use, so you can get started with Athens without even installing it. For details, see [here](./public_proxy).
+We also run a public version of Athens so you can get started without even installing anything. To get started, set `GOPROXY="https://athens.azurefd.net"`.
```

Comment:
@chriscoffee let's keep this as an experimental server because it's pretty unstable now and we don't have any policies we'd need to run this as an official server (e.g. DMCA takedowns, security vulnerabilities, ...)

Reply:
resolved in https://github.com/gomods/athens/pull/964/commits/e87725d7d2ad20da828be9a7a5480f9f9589b502